about.html

```diff
@@ -0,0 +1,102 @@
+      </ul>
+    </div>
+    <header id="homepage">
+      <div class="top-header">
+        <ul class="top-nav-list">
+          
+        </ul>
+      </div>
+      <nav id="nav-bar-mobile" class="nav-bar-mobile">
+        <div class="mobile-nav">
+          <a id="hamburger" href="#"><i class="fa-solid fa-bars menu-icon"></i></a>
+        </div>
+        <div id="mobile-menu" class="mobile-menu inactive">
+          <div id="closebutton" class="close-btn"><a href="#"><i class="fa-solid fa-xmark"></i></a></div>
+          <ul>
+            <li><a id="button1" href="index.html">Home</a></li>
+            <li><a id="button2" href="#past-events">Past Events</a></li>
+            <li><a id="button3" href="#our-mission">Our Mission</a></li>
+            <li><a id="button4" href="#partners">Partners</a></li>
+          </ul>
+        </div>
+      </nav>
+
+      <div class="hero-content-about">
+        <h1>About Us</h1>
+        <p>At Vibetune, we're passionate about bringing people together through music. Our team of experienced professionals is dedicated to organizing and promoting unforgettable concert experiences that showcase a diverse range of musical genres and artists. With a focus on high-quality sound, immersive visuals, and top-notch production, we strive to create an atmosphere that celebrates the power of music to connect and inspire. Join us for the ultimate concert experience - let's feel the vibe together.</p>
+      </div>
+    </header>
+
+    <section id="our-mission" class="our-mission">
+        <h2>Our Mission</h2>
+        <p>At Vibetune, our mission is to bring people together through music. We believe that music has the power to inspire, uplift, and connect people from all walks of life. That's why we're committed to creating unforgettable concert experiences that celebrate a diverse range of musical genres and artists.
+
+            We're also dedicated to promoting diversity and inclusion in the music industry. We believe that music should be a platform for all voices to be heard and celebrated, regardless of race, gender, or any other characteristic. That's why we strive to work with artists from all backgrounds and to create an environment that is welcoming and inclusive to all.
+            
+            We're also committed to making a positive impact on the environment and our communities. That's why we use sustainable practices in our event production and partner with social and environmental organizations to promote positive change.
+            
+            Join us in our mission to celebrate the power of music to connect and inspire</p>
+
+    </section>
+
+    <section id="past-events" class="past-events">
+        <h2>Past Events</h2>
+        <div class="events-container">
+            <div class="events1">
+              <h3>Coldplay Live in Concert, 2017</h3>
+            </div>
+            <div class="events2">
+              <h3>Harry Styles Live in Concert, 2018</h3>
+            </div>
```

Comment:
- [ ] Again let's try as much as possible to stick to the original design as much as possible. You should make sure that the texts on top of the background images are in order of the `year` above and the `description text` below it. These different texts should also have varying sizes ☺️.